### PR TITLE
Obscure Admin Related fixes

### PR DESCRIPTION
**Pull request by Aurorablade**
I am keeping the typo tiger...

### CODE REVIEW

File: code/modules/admin/IsBanned.dm [hunk 1]
```diff
@@ -11,6 +11,9 @@ world/IsBanned(key,address,computer_id)
 	var/ckey = ckey(key)
 	if((ckey in admin_datums) || (ckey in deadmins))
 		admin = 1
+	var/datum/admins/A = admin_datums[ckey]
```
Comment (tigercat2000) [inline code/modules/admin/IsBanned.dm]: this should be inside the if above and delete L13

File: code/modules/admin/topic.dm [hunk 2]
```diff
@@ -887,7 +887,8 @@
 	else if(href_list["boot2"])
 		var/mob/M = locate(href_list["boot2"])
 		if(ismob(M))
-			if(!check_if_greater_rights_than(M.client))
+			if(M.client && M.client.holder && (M.client.holder.rights & R_BAN))
+				to_chat(src, "you acnt kick this fucker")
```
Comment (tigercat2000) [inline code/modules/admin/topic.dm]: Make it a proper error message please :stuck_out_tongue:

File: code/modules/admin/topic.dm [hunk 3]
```diff
@@ -888,7 +888,7 @@
 		var/mob/M = locate(href_list["boot2"])
 		if(ismob(M))
 			if(M.client && M.client.holder && (M.client.holder.rights & R_BAN))
-				to_chat(src, "you acnt kick this fucker")
```
Comment (KasparoVy) [inline code/modules/admin/topic.dm]: RIP, made me laugh proper

Comment (taukausanake) [reply code/modules/admin/topic.dm]: Press f to pay respects

**Pull request merged**